DEVELOPER.md

```diff
@@ -0,0 +1,374 @@
+# Developer Guide for TestSys
+
+## What is TestSys
+
+TestSys is a framework that leverages Kubernetes (k8s) to create resources and run tests.
+To learn more about how it works, see the [design](design/DESIGN.md) document.
+
+### Navigating the `bottlerocket-test-system` repo
+
+#### [Makefile.toml](Makefile.toml)
```

Comment:
This should be `Makefile` since we don't use `cargo-make` in testsys (should we?)